CI/hf_mirror/download_hf_models.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+df -h  # debug purpose to make sure efs is attached correclty
+git clone https://github.com/autogluon/autogluon.git
+python3 download_hf_models.py --model_list_file autogluon/multimodal/tests/hf_model_list.yaml
```

Comment:
Wondering shall we even allow users to pass a separate file list to the python function as it might expose us to security risks. If there is no such use case, we shall go with the hard coded solution

Reply:
Yea I get your point. Though this script is actually inside the docker container hence cannot be modified unless re-pushed. Therefore, we will always pass `autogluon/multimodal/tests/hf_model_list.yaml` and user cannot modify it